app/src/main/res/values/preference.xml

```diff
@@ -131,4 +131,6 @@
     <string name="pref_export_certificate_summary">Exports a certificate to external storage.</string>
     <string name="pref_show_user_count">Show Channel User Count</string>
     <string name="pref_show_user_count_summary">Show a counter with the number of users in a channel subtree.</string>
+    <string name="pref_startUpInPinnedMode">Start Up In Pinned Mode</string>
```

Comment:
Convention for (most) prefs is to separate words with underscores, not camelCase.